### ISSUE TITLE
Single pointer option is not provided for rearranging the "Add a trigger" control.

### ISSUE THREAD

**Issue by Xiaoyu-Huang** [Azure/LogicAppsUX#5275]
### Describe the issue 

Repro step:

 navigate to add a trigger when creating new workflow

 Actual Result:

 Single pointer option is not provided for rearranging the "Add a trigger" control.

 Observation: "Add a trigger" control is rearrange using mouse

 Expected Result:

 A single pointer option should be provided for rearranging the "Add a trigger" control.

### Are you using new designer or old designer 

New Designer (With action panels)

### Screenshots or Videos 

[https://github.com/user-attachments/assets/52b0c52a-387a-4395-b726-ef1c762fbbb6](<https://github.com/user-attachments/assets/52b0c52a-387a-4395-b726-ef1c762fbbb6>)

### Browser 

Edge

### Additional context 

*No response*

**Comment by hartra344** [comment]
I'm not sure I understand exactly what's being looked for here? Using a stylus or other input option to move the "add a trigger" button around?

**Comment by Xiaoyu-Huang** [comment]
> I'm not sure I understand exactly what's being looked for here? Using a stylus or other input option to move the "add a trigger" button around?

my understanding is, we could use mouse to move the add a trigger on canvas. but we are not able to use keyboard to move it

**Comment by rllyy97** [comment]
When you drag the "Add a trigger" node you are actually just dragging the entire canvas like if you drag the whitespace around the node.  Currently we have no way of moving nodes with the keyboard, but you should be able to navigate to all nodes on the canvas using normal tab navigation.
Closing this issue for now
